doc/source/groupby.rst

```diff
@@ -1165,6 +1165,54 @@ See the :ref:`visualization documentation<visualization.box>` for more.
   to ``df.boxplot(by="g")``. See :ref:`here<visualization.box.return>` for
   an explanation.
 
+.. _groupby.pipe:
+
+Piping function calls
+~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.21.0
+
+Similar to the functionality provided by ``DataFrames`` and ``Series``, functions
```

Comment:
DataFrames -> DataFrame (or otherwise do not put it as a code object)

Reply:
ok